Track.h

```diff
@@ -570,6 +573,7 @@ inline int getScoreCalc(const unsigned int seedtype,
   */
   ////// V1 of candidate score (after fix for counts of # missing hits):
   // For high pT central tracks: 4x valid hit bonus and 0.25x missing hit penalty
+  /*
```

Comment:
instead of commenting out a better practice would be to allow to choose with a switch
or simply remove if versioning is not to be really supported

Reply:
When doing the tuning, it was useful that @mmasciov left V0 in the code but commented out, so that I could easily see what changed from V0 to V1. But that could also be checked in the github history, so I'm fine with removing the V0 and V1 comment blocks if @mmasciov agrees.